tests/drivers/pwm/pwm_api/src/test_pwm.c

```diff
@@ -81,7 +81,7 @@
 #define DEFAULT_PWM_PORT 2 /* TCC1/WO[2] on PA18 (D7) */
 #elif defined CONFIG_BOARD_MIMXRT685_EVK
 #define DEFAULT_PWM_PORT 7 /* D3 on Arduino connector J27 */
-#elif defined CONFIG_BOARD_LPCXPRESSO55S69_LPC55S69_CPU0
+#elif defined CONFIG_BOARD_LPCXPRESSO55S69_LPC55S69_CPU0_NS
```

Comment:
Is there anything that keeps this code from working on either, secure or non-secure builds? If not, I'd rather change this to the following:

```suggestion
#elif defined(CONFIG_BOARD_LPCXPRESSO55S69_LPC55S69_CPU0_NS) ||                                    \
	defined(CONFIG_BOARD_LPCXPRESSO55S69_LPC55S69_CPU0)
```

Reply:
Updated accordingly. I think this is correct now.